sklearn/mixture/_bayesian_mixture.py

```diff
@@ -255,6 +255,10 @@ class BayesianGaussianMixture(BaseMixture):
         Lower bound value on the model evidence (of the training data) of the
         best fit of inference.
 
+    lower_bounds_ : array-like of shape (`n_iter_`,)
+        The list of lower bound values at each iteration of the best fit of
+        EM algorithm.
+
```

Comment:
```
lower_bounds_ : array-like of shape (`n_iter_`,)
    The list of lower bound values on the model evidence from each iteration of the best fit of inference.
```
Thoughts on aligning with lower_bound_ for consistency?

Reply:
This makes total sense! I modified both descriptions. Thanks you for your review!